src/ENSMetadata.sol

```diff
@@ -77,8 +77,6 @@ contract ENSMetadata {
         // If verification passes, set verification status to true
         metadata.verification = true;
```

Comment:
If we agree, this should be fixed by previous comment(need check for success or failure)